src/components/ButtonPanel.js

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
+import Button from './Button';
 
 export default function ButtonPanel() {
-  return <p>Future Calculator</p>;
+  return (
+    <table>
+      <tr>
+        <Button name="AC" col={1} />
+        <Button name="+/-" col={1} />
+        <Button name="%" col={1} />
+        <Button name="/" col={1} />
```

Comment:
[Optional] We can assign a default value of 1 to the col prop in the Button component. so that you don't have to send repeated col={1} prop to most of the Button components.